### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by JamesH65**
Clarify how the USB2.0 ports are connected on the Pi4.

### CODE REVIEW

**Comment by lurch** [comment]
I wonder if it's worth clarifying that this also applies to USB2.0 devices plugged into the Pi4's USB3.0 ports? (which I only know because I've just read the linked issue https://github.com/raspberrypi/linux/issues/3406 )

**Comment by pelwell** [comment]
I think you could delete `. Therefore all USB 2.0 devices share this one bus's bandwidth`, reducing the repetition without losing any information.

**Comment by JamesH65** [comment]
Had a go at fixing both previous comments....thoughts on a postcard please.

File: hardware/raspberrypi/usb/README.md [hunk 1]
```diff
@@ -28,9 +28,11 @@ OTG in general supports communication to all types of USB device, but to provide
...
+For the Pi 4, a fully-featured host controller drives the downstream USB ports. Downstream USB is provided by a Via Labs VL805 chip - that supports two USB 2.0 ports and two USB 3.0 ports. This is connected to the BCM2711 SoC using a PCIe link, which is extremely fast. Therefore, the Pi 4 does not have the same speed constraints of previous models, which means very fast data transfer rates, especially when using the USB 3.0 ports. All connected USB 2.0 devices are connected via an internal hub which connects to the upstream PCIe via a single USB 2.0 bus, giving a maximum combined bandwith for all USB 2.0 devices of 480Mbits/s. All four USB ports on the device are connected to the USB 2.0 hub, whilst the USB 3.0 ports (blue) are ALSO connected to the USB 3.0 port via the USB3.0 specific pins in the socket. USB3.0 devices are constrained only by the total bandwidth available over the PCIe link.
```
Comment (lurch) [inline hardware/raspberrypi/usb/README.md]: Just some little nitpicks:
 * should "connects to the upstream PCIe" be "connects to the upstream PCIe link" ?
 * small inconsistency on whether there's a space displayed or not in "USB 3.0" or "USB3.0"
 * "the USB 3.0 ports (blue) are ALSO connected to the USB 3.0 port" doesn't sound right - should the last "port" in that phrase be "bus" or something similar?

File: hardware/raspberrypi/usb/README.md [hunk 2]
```diff
@@ -28,9 +28,11 @@ OTG in general supports communication to all types of USB device, but to provide
...
 
-It also means that almost all of the known issues with USB on previous models are no longer present.
+You can use `lsusb -t` to display how the USB devices and hubs are arranged and their allocated speeds. 
+
+Most of the technical limitations of the USB on previous models are no longer present.
```
Comment (lurch) [inline hardware/raspberrypi/usb/README.md]: I'm not sure if "technical limitations of the USB" sounds right either?
"technical limitations of the USB ports" ? "technical limitations of the USB implementation" ? :man_shrugging:

**Comment by JamesH65** [comment]
OK, fixed up @lurch final comments - don't think it's worth spending too much more time on this, so will merge in a  hours or so unless anyone has any big objections.

**Pull request merged**